variants/STM32G0xx/G071C(6-8-B)(T-U)_G081CB(T-U)/variant_agafia_sg0.h

```diff
@@ -0,0 +1,167 @@
+#define PD1  23
+#define PD2  24
+#define PD3  25
+#define PD4  26 
+#define PB4	 27 	
+#define PB5  28
+#define PB6  29 
+#define PB7  30
+
+#define PA14 31 //SCK-BOOT0
+#define PA13 32 //SWDIO
+
+#define PF0  33 //LED3 
+#define PF1  34 //LED2
+#define PC13 35 //LED1
+
+#define PA4  36 // A0
+#define PB12 37 // A1
+#define PB11 38 // A2
+#define PB10 39 // A3
+#define PB2  40 // A4
+#define PB1  41 // A5
+
+// This must be a literal
+#define NUM_DIGITAL_PINS        42
+// This must be a literal with a value less than or equal to to MAX_ANALOG_INPUTS
+#define NUM_ANALOG_INPUTS       10
+ 
+
+// On-board LED pin number
+#ifndef LED_BUILTIN
+  #define LED_BUILTIN           PC13
+#endif 
+#define LED1                    LED_BUILTIN
+#define LED2                    PF0  
+#define LED3                    PF1
+ 
+
+// SPI Definitions
+#define PIN_SPI_SS              PB0
+#define PIN_SPI_MOSI            PA7
+#define PIN_SPI_MISO            PA6
+#define PIN_SPI_SCK             PA5
+
+// I2C Definitions
+#define PIN_WIRE_SDA            PB9
+#define PIN_WIRE_SCL            PB8
+
+// Timer Definitions
+// Use TIM6/TIM7 when possible as servo and tone don't need GPIO output pin
+#ifndef TIMER_TONE
+  #define TIMER_TONE            TIM6
+#endif
+#ifndef TIMER_SERVO
+  #define TIMER_SERVO           TIM7
+#endif
+
+#define PIN_DAC1                PA4
+#define PIN_DAC2                PA5
+
+// UART Definitions
+#ifndef SERIAL_UART_INSTANCE
+  #define SERIAL_UART_INSTANCE  101 //Connected to ST-Link
+#endif
+
+// Default pin used for 'Serial' instance (ex: ST-Link)
+// Mandatory for Firmata
+#ifndef PIN_SERIAL_RX
+  #define PIN_SERIAL_RX         PA3
+#endif
+#ifndef PIN_SERIAL_TX
+  #define PIN_SERIAL_TX         PA2
+#endif
+
+// Extra HAL modules
+#if !defined(HAL_DAC_MODULE_DISABLED)
+  #define HAL_DAC_MODULE_ENABLED
+#endif
+
+ 
+/*----------------------------------------------------------------------------
+ *        Arduino objects - C++ only
+ *----------------------------------------------------------------------------*/
+
+#ifdef __cplusplus
+// These serial port names are intended to allow libraries and architecture-neutral
+// sketches to automatically default to the correct port name for a particular type
+// of use.  For example, a GPS module would normally connect to SERIAL_PORT_HARDWARE_OPEN,
+// the first hardware serial port whose RX/TX pins are not dedicated to another use.
+//
+// SERIAL_PORT_MONITOR        Port which normally prints to the Arduino Serial Monitor
+//
+// SERIAL_PORT_USBVIRTUAL     Port which is USB virtual serial
+//
+// SERIAL_PORT_LINUXBRIDGE    Port which connects to a Linux system via Bridge library
+//
+// SERIAL_PORT_HARDWARE       Hardware serial port, physical RX & TX pins.
+//
+// SERIAL_PORT_HARDWARE_OPEN  Hardware serial ports which are open for use.  Their RX & TX
+//                            pins are NOT connected to anything by default.
+#define SERIAL_PORT_MONITOR   Serial
+#define SERIAL_PORT_HARDWARE  Serial2
```

Comment:
Based on your declaration:
https://github.com/stm32duino/Arduino_Core_STM32/pull/1690/files#diff-240fd52966412d67ec0d5122615cf85a79f0adcf9951f38508616f7a56b4923dR126

it should be:
```suggestion
#define SERIAL_PORT_HARDWARE  SerialLP1
```
or simply use `Serial` which is mapped on it.